libraries/PDM/src/PDM.cpp

```diff
@@ -116,7 +128,6 @@ int PDMClass::begin(int channels, long sampleRate)
   nrf_pdm_event_clear(NRF_PDM_EVENT_STARTED);
   nrf_pdm_task_trigger(NRF_PDM_TASK_START);
```

Comment:
The register buffer pointer is still set to `nullptr`.  Can you help me understand why this might be intentional?